src/Login.ts

```diff
@@ -162,12 +162,13 @@ export class Login {
 
     // Step 2
     const authTokens = Login.parseQuery(window.location.search); // this line can throw
+    // Step 2.1
+    let url = window.location.href;
+    url = removeParameterFromUrl(url, ACCESS_TOKEN);
+    url = removeParameterFromUrl(url, ID_TOKEN);
+    window.history.replaceState(null, '', url);
```

Comment:
What if we passed in tokens and returned on the step 1.2? Should we also remove tokens from url?

Reply:
If that is the case then we should ideally not ever have any id_token / access_token in the URL :)